README.adoc

```diff
@@ -32,146 +30,43 @@ Existing tools such as `oc` are more operations-focused and require a deep-under
 * Detects changes to local code and deploys it to the cluster automatically, giving instant feedback to validate changes in real-time.
 * Lists all available components and services from the OpenShift cluster.
 
-[[installing-odo]]
-== Installing `odo`
-
-=== Prerequisites:
-
-OpenShift 3.10.0 or above is required.
-
-=== Procedure:
-
-==== Mac and Linux
-
-===== OS-independent automated installation for Mac and Linux systems
-
-Use this link:./scripts/installer.sh[bash script] to quickly install `odo`. It will automatically detect your operating system and install `odo` accordingly.
+[[official-documentation]]
+== Official documentation
 
-----
-$ curl -L https://github.com/openshift/odo/raw/master/scripts/installer.sh | bash
-----
+* link:https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_developer_cli/installing-odo.html[Installing odo]
+* link:https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_developer_cli/creating-a-single-component-application-with-odo.html[Creating a single-component application with odo]
+* link:https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_developer_cli/creating-a-multicomponent-application-with-odo.html[Creating a multicomponent application with odo]
+* link:https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_developer_cli/creating-an-application-with-a-database.html[Creating an application with a database]
+* link:https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_developer_cli/configuring-the-odo-cli.html[Configuring the odo CLI]
+* link:https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_developer_cli/odo-cli-reference.html[odo CLI reference]
+* link:https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_developer_cli/odo-release-notes.html[odo release notes]
 
-===== Binary installation for Mac OS
-
-----
-# curl -L https://github.com/openshift/odo/releases/latest/download/odo-darwin-amd64 -o /usr/local/bin/odo && chmod +x /usr/local/bin/odo
-----
-
-===== Binary installation for Linux
-
-----
-# curl -L https://github.com/openshift/odo/releases/latest/download/odo-linux-amd64 -o /usr/local/bin/odo && chmod +x /usr/local/bin/odo
-----
-
-For more installation options please refer the link:docs/installation.adoc[installation page].
+[[installing-odo]]
 
-==== Windows
+== Installing `odo`
```

Comment:
Do we need this section if there is a link to "Installing odo" above?

Reply:
Yes, or else it may be missed since it's an index / table of contents. Better to indicate where exactly how to install as well as the mirror links.

Reply:
ok, makes sense

/approve